### PR TITLE
Fix regression with exotic timestamp

### PR DESCRIPTION
**Pull request by sileht**
We recently uses local TZ, when timestamps doesn't have a TZ.
But Gnocchi client was passing the string as-is before allowing
all kind of timestamps that the server support (like +2hour, now, ...).

This change restores this feature.

### CODE REVIEW

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 3a482a2ed2b4fd0a8037ac591a1a4319e8d4c40d

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 7ba17d2cc8fdcfc2eccab59421d541c3833bcfa2

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 7ba17d2cc8fdcfc2eccab59421d541c3833bcfa2

**Comment by sileht** [comment]
Pull-request updated, HEAD is now c1b4b7593e83e46fe51d6408364e2ba7788fe225

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 72b941d132f9a03f2920d126e1b7381933888283

File: gnocchiclient/tests/functional/test_metric.py [hunk 1]
```diff
@@ -77,6 +77,31 @@ def test_metric_listing(self):
                               u" --limit 2")
         self.assertEqual(2, len(json.loads(result)))
 
+    def test_metric_measures_exhautic_format(self):
```
Comment (chungg) [inline gnocchiclient/tests/functional/test_metric.py]: exotic

**Pull request merged**